packager/react-packager/src/DependencyResolver/fastfs.js

```diff
@@ -293,16 +293,16 @@ class File {
   }
 
   getFiles() {
-    const files = [];
-    Object.keys(this.children).forEach(key => {
+    const files = Object.keys(this.children).map(key => {
       const file = this.children[key];
       if (file.isDir) {
-        files.push(...file.getFiles());
-      } else {
-        files.push(file);
+        return file.getFiles();
```

Comment:
I think it might be more efficient to bring back what we had previously but on this line we'd use `files = files.concat(file.getFiles())`.